meta-mender-commercial-demo/recipes-extended/images/mender-demo-delta.bb

```diff
@@ -0,0 +1,6 @@
+require recipes-extended/images/mender-demo-minimal-rw.bb
+
+IMAGE_INSTALL_append = " mender-binary-delta"
+
+IMAGE_FEATURES_append = " read-only-rootfs"
+EXTRA_IMAGECMD_ext4_append = "${@bb.utils.contains('IMAGE_FEATURES', 'read-only-rootfs', ' -O ^64bit -O ^has_journal', '', d)}"
```

Comment:
I'm not quite clear on what this layer adds compared to the original one. What is the intended use for this layer?

Reply:
You're right. It provides almost nothing. Will rethink this.